pkg/controller/mongodb/statefulset.go

```diff
@@ -217,11 +221,12 @@ spec:
               mountPath: /work-dir
             - name: tmp-mongodb
               mountPath: /tmp
-
         - name: metrics
           image: "{{ .ImageRepo }}/{{ .MetricsImage }}"
           imagePullPolicy: "IfNotPresent"
           securityContext:
+            runAsUser: 999
```

Comment:
Have you test with this PR. I am not sure if the operator service account can use user 999

Reply:
no, I'm still working on getting the new image I can test...I don't think prow is set up to send the build anywhere, I atleast cannot find it in the logs